### PR TITLE
Fix scope constant in introspection module

### PR DESCRIPTION
**Pull request by scymtym**
Looks like it was damaged during the "Replace double underscores with a single one" change (954ee8b).

### CODE REVIEW

**Comment by scymtym** [comment]
Added basic test. Pretty sure it will not survive the style checker.

**Comment by languitar** [comment]
> Added basic test. Pretty sure it will not survive the style checker.

You can test that beforehand with `tox -e check`, assuming that pyenv has been set up as described in the readme.

File: test/test_rsb_introspection.py [hunk 1]
```diff
@@ -55,3 +59,26 @@ def test_construction_defaults(self):
         assert isinstance(info.machine_type, str)
         assert isinstance(info.software_type, str)
         assert isinstance(info.software_version, str)
+
+
+class TestIntrospectionEvents:
+
+    @pytest.mark.usefixture('rsb_config_socket')
+    def test_introspection_events(self):
+        # The listener must be able to deserialize introspection
```
Comment (languitar) [inline test/test_rsb_introspection.py]: Could you please rewrap comments at 80 characters. This is used throughout the whole source code.

File: test/test_rsb_introspection.py [hunk 2]
```diff
@@ -55,3 +59,26 @@ def test_construction_defaults(self):
...
+
+        events = queue.Queue()
+        with rsb.create_listener(rsb.introspection.BASE_SCOPE) as listener:
+            listener.add_handler(events.put)
+            # Create and destroy a participant with enabled
+            # introspection. This generates two introspection events.
+            rsb.get_default_participant_config().introspection = True
+            with rsb.create_informer('/foo'):
+                pass
+            while events.qsize() < 2:
```
[the remark this answers is not in the document]
Comment (scymtym) [reply test/test_rsb_introspection.py]: Done.

File: test/test_rsb_introspection.py (hunk 1 above)
Comment (scymtym) [reply test/test_rsb_introspection.py]: Done.

**Comment by scymtym** [comment]
Still needs rebase, will do that.

**Comment by languitar** [comment]
Seems the codecov report got updated here.

File: test/test_rsb_introspection.py [hunk 3]
```diff
@@ -55,3 +59,33 @@ def test_construction_defaults(self):
...
+        # the initialization of the introspection module.
+        rsb._initialize_introspection()
+
+        # Collect events in a thread-safe way and make sure the number is right.
+        events = []
+        events_cv = threading.Condition()
+        def add_event(event):
+            with events_cv:
+                events.append(event)
+                if len(events) == 2:
```
Comment (languitar) [inline test/test_rsb_introspection.py]: There's still a bit of redundancy here with all the comparisons to `2`. Maybe at least a variable that explains the meaning for that value, which is used in all places, would be good

Comment (scymtym) [reply test/test_rsb_introspection.py]: Done.

**Pull request merged**